Dockerfile

```diff
@@ -19,28 +16,39 @@ ENV VUE_APP_API_URL=https://datacube.zazukoians.org/
 ENV NODE_ENV=production
 RUN npm run build
 
-WORKDIR /api
-ADD api ./
+FROM node:lts-alpine AS build-api
+
+WORKDIR /
+ADD api ./api/
+ADD packages/rdfine-csvw ./packages/rdfine-csvw/
+ADD packages/rdfine-data-cube ./packages/rdfine-data-cube/
+ADD lerna.json .
+ADD package*.json ./
 # install and build backend
 ENV NODE_ENV=
-RUN npm ci
+RUN npm ci --unsafe-perm
 ENV NODE_ENV=production
 RUN npm run build
 
 # Second step: only install runtime dependencies
 FROM node:lts-alpine
 
-WORKDIR /api
+WORKDIR /app
 
-RUN npm i -g nodemon ts-node typescript
+RUN npm i -g nodemon ts-node typescript lerna
 
-COPY --from=builder /api/package.json ./package.json
-COPY --from=builder /api/package-lock.json ./package-lock.json
-RUN npm ci --only=production
+COPY --from=build-api /lerna.json ./lerna.json
+COPY --from=build-api /package*.json ./
+COPY --from=build-api /api/package*.json ./api/
+COPY --from=build-api /packages/rdfine-csvw/package*.json ./packages/rdfine-csvw/
+COPY --from=build-api /packages/rdfine-data-cube/package*.json ./packages/rdfine-data-cube/
+RUN lerna bootstrap -- --only=production
```

Comment:
Maybe a stupid question but since we're building the package in a separate step, why do we need to bootstrap again here?

Reply:
The build step installs all dependencies. Here we copy the build output and the install only `dependencies` and not `devDependencies`.

Reply:
Ok, I though "building" would also bundle any required dependency. I don't understand node very well yet ;)